raster/r.out.gdal/r.out.gdal.html

```diff
@@ -398,8 +398,3 @@ <h2>AUTHORS</h2>
 
 Vytautas Vebra (oliver4grass at gmail.com)<br>
 Markus Metz (improved nodata logic)
-
```

Comment:
SEE ALSO section should be after REFERENCES section.

Reply:
fixed